tests/pull_from_remote_test.py

```diff
@@ -0,0 +1,126 @@
+import os
+import shutil
+import subprocess
+import random
+import string
+import unittest
+from gitautosync import pull_from_remote
+
+
+class TestPullFromRemote(unittest.TestCase):
```

Comment:
use https://docs.pytest.org/en/latest/getting-started.html rather than the unittest module. py.test is much cleaner and is generally much more widely used now.